tests/ui/author/struct.rs

```diff
@@ -1,13 +1,11 @@
-#[allow(clippy::unnecessary_operation, clippy::single_match)]
+#![allow(clippy::unnecessary_operation, clippy::single_match, clippy::no_effect)]
 fn main() {
     struct Test {
         field: u32,
     }
 
     #[clippy::author]
-    Test {
-        field: if true { 1 } else { 0 },
-    };
```

Comment:
Haven't looked at the PR in detail yet. Could you revert this and allow the new lint in this file? (Or alternatively just use different number literals?)

Reply:
Fixed in [26468a8](https://github.com/rust-lang/rust-clippy/pull/9086/commits/26468a814936662a14ea1500dea4ba970def0646)